audio/buffer_source_node.rs

```diff
@@ -8,6 +8,8 @@ use param::{Param, ParamType};
 pub enum AudioBufferSourceNodeMessage {
     /// Set the data block holding the audio sample data to be played.
     SetBuffer(Option<AudioBuffer>),
+    /// Set start parameters (when, offset, duration).
+    Start(f64, Option<f64>, Option<f64>),
```

Comment:
nit: rename this to `SetStartParams` or such, so it is not confused with the `AudioScheduledSourceNodeMessage::Start` message.

It would also be nice to remove the first parameter (`when`) as we can get it from `AudioScheduledSourceNodeMessage::Start`.

Reply:
I fixed the first part, and will leave the second part to the AudioScheduledSourceNode follow-up.